main.tf

```diff
@@ -143,36 +143,55 @@ data "aws_ami" "user_ami" {
   }
 }
 
-resource "aws_launch_configuration" "container_instance" {
-  lifecycle {
-    create_before_destroy = true
+resource "aws_launch_template" "container_instance" {
+  block_device_mappings {
+    device_name = "/dev/sda1"
```

Comment:
![image](https://user-images.githubusercontent.com/1774125/41617110-a6bc876c-73cd-11e8-9640-a844b9c4ba75.png)

https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/block-device-mapping-concepts.html

Reply:
This seems like the correct value.